File: dataset/javascript_mix/C06WordcountType1A.js
function wordCount(s) {
    const counts = {};
    const words = s.trim().split(/\s+/);
    for (const w of words) {
        counts[w] = (counts[w] ?? 0) + 1;
    }
    return counts;
}

module.exports = { wordCount };

File: test/java/mix/C06WordcountType1A.java
import java.util.HashMap;
import java.util.Map;

public class C06WordcountType1A {
    public static Map<String, Integer> wordCount(String s) {
        Map<String, Integer> counts = new HashMap<>();
        for (String w : s.split("\\s+")) {
            counts.put(w, counts.getOrDefault(w, 0) + 1);
        }
        return counts;
    }
}
